Systemuser 2.11

diocas opened the pull request:
- New extra save for Jupyter notebooks. 
  - This keeps a local copy of notebooks to be able to recover them in case of session failure.
- Nbextensions deployment. 
  - This allows us to deploy nbextensions from the local machine.

etejedor commented on systemuser.sh:
```diff
@@ -28,7 +32,7 @@ JPY_LOCAL_DIR=$SCRATCH_HOME/.local
 export JUPYTER_CONFIG_DIR=$JPY_DIR
 # Our kernels will be in $JPY_LOCAL_DIR/share/jupyter, $LCG_VIEW/share/jupyter is needed for the notebook extensions
-export JUPYTER_PATH=$JPY_LOCAL_DIR/share/jupyter:$LCG_VIEW/share/jupyter
+export JUPYTER_PATH=$JPY_LOCAL_DIR/share/jupyter:$LCG_VIEW/share/jupyter$EXTRA_LIBS
```
What is this $EXTRA_LIBS in the JUPYTER_PATH for?
Also, shouldn't there be a ':' before $EXTRA_LIBS? Even if the ':' is already inside $EXTRA_LIBS, it would be cleaner to have it outside.

diocas replied:
$EXTRA_LIBS is a path pointing to a folder that, in the future, will have SWAN nbextensions. Like the LCG ones. 
I've put the ':' inside $EXTRA_LIBS, yes. I will move them here.

etejedor replied:
Ok thanks for the explanation. However, since this PR (and commit) is about the extra save, I believe this particular change should not belong to this PR.

diocas replied:
Do you want me to make a new PR with just this line? I joined these two things because this PR is not just for "extra save", is for "v2.11" (that contains extra save + notifications nbextensions).

etejedor replied:
I would at least divide it in two commits, each with its own message :smile: 
I don't know if you can edit now the description of the PR, if possible please introduce some text there as well, thanks!

diocas committed:
Extra save in the main docker image
Keep a local copy of notebooks to prevent users from loosing their work if something goes wrong with their session.

diocas committed:
Added nbextensions deployment method
New path added to JUPYTER_PATH to get SWAN extensions from local machine.
